### DISCUSSION TITLE
Timestamps in logs - GSOC 2024

### DISCUSSION THREAD

**Discussion by pratik-choudhari** [sosy-lab/benchexec#1024] [General]
Hello @PhilippWendler,
I am Pratik Choudhari and I am looking into the **Timestamps in logs** idea for GSOC 2024. Based on the existing discussions, I have already started reading the paper and running benchexec on my Ubuntu 20.04 VM. Will put my questions in this discussion thread, thanks!

**Comment by PhilippWendler** [comment]
Welcome to the BenchExec project!

If you want to participate in GSoC it will be highly beneficial to familiarize yourself with our code base already during the proposal writing and try to tackle and issue and file a PR to demonstrate that you can successfully contribute.
